docs/topics/ksp/ksp-incremental.md

```diff
@@ -4,55 +4,56 @@ Incremental processing is a processing technique that avoids re-processing of so
+To enable logs that dump the dirty set according to dependencies and outputs, use `ksp.incremental.log=true`. 
+You can find these log files in the `build` output folder with a `.log` file extension.
 
-## Aggregating vs Isolating
+## Aggregating vs Isolating
 
 Similar to the concepts in [Gradle annotation processing](https://docs.gradle.org/current/userguide/java_plugin.html#sec:incremental_annotation_processing),
-KSP supports both _aggregating_ and _isolating_ modes. Note that unlike Gradle annotation processing,
-KSP categorizes each output as either aggregating or isolating, rather than the entire processor.
+KSP supports both _aggregating_ and _isolating_ modes. Note that unlike Gradle annotation processing, KSP categorizes 
+each output as either aggregating or isolating, rather than the entire processor.
 
-An aggregating output can potentially be affected by any input changes, with the exception of removing files that don't affect
+An aggregating output can potentially be affected by any input changes, except removing files that don't affect
 other files. This means that any input change results in a rebuild of all aggregating outputs,
-which in turn means that all of the corresponding registered, new, and modified source files are reprocessed.
+which in turn means reprocessing of all corresponding registered, new, and modified source files.
 
-As an example, an output that collects all symbols with a particular annotation is considered an `aggregating` output.
+As an example, an output that collects all symbols with a particular annotation is considered an aggregating output.
 
-An _isolating_ output depends only on its specified sources. Changes to other sources do not affect an isolating output.
+An isolating output depends only on its specified sources. Changes to other sources do not affect an isolating output.
 Note that unlike Gradle annotation processing, you can define multiple source files for a given output.
 
-As an example, a generated class that is dedicated to an interface it implements is considered *isolating*.
+As an example, a generated class that is dedicated to an interface it implements is considered isolating.
 
-To summarize, if an output might depend on new or any changed sources, it is considered *aggregating*. Otherwise, the output is *isolating*.
+To summarize, if an output might depend on new or any changed sources, it is considered aggregating. 
+Otherwise, the output is isolating.
 
 Here's a summary for readers familiar with Java annotation processing:
-* In an _isolating_ Java annotation processor, all the outputs are _isolating_ in KSP.
-* In an _aggregating_ Java annotation processor, some outputs can be _isolating_ and some be
-_aggregating_ in KSP.
+* In an isolating Java annotation processor, all the outputs are isolating in KSP.
+* In an aggregating Java annotation processor, some outputs can be isolating and some can be aggregating in KSP.
 
 ## Example 1
```

Comment:
I expect better wording in the title. Example of...

Reply:
The examples are almost about the same, so it's a bit complicated to name each one. I came up with another solution, please see [the commit](https://github.com/JetBrains/kotlin-web-site/pull/2570/commits/9644e5e19fcc3f6f697073aacf4139c769ae9382).

Reply:
I prefer two separated examples, TBH.